src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -4,41 +4,24 @@
 import com.github.lindenb.jbwa.jni.BwaIndex;
 import com.github.lindenb.jbwa.jni.BwaMem;
 import com.github.lindenb.jbwa.jni.ShortRead;
-import org.broadinstitute.hellbender.utils.NativeUtils;
 import org.broadinstitute.hellbender.utils.bwa.BWANativeLibrary;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
 import org.testng.Assert;
-import org.testng.SkipException;
 import org.testng.annotations.Test;
 
 import java.io.File;
 import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
 
 public class JBWAIntegrationTest extends BaseTest {
 
-    private void skipJBWATestOnUnsupportedPlatforms() {
-        if ( ! NativeUtils.runningOnLinux() && ! NativeUtils.runningOnMac() ) {
-            throw new SkipException("jbwa not available on this platform");
-        }
-        if ( NativeUtils.runningOnPPCArchitecture() ) {
-            throw new SkipException("jbwa not available for this architecture");
-        }
-    }
-
     @Test
     public void testJBWAIsLoadable(){
-        skipJBWATestOnUnsupportedPlatforms();
-
         BWANativeLibrary.load();
     }
```

Comment:
I think it's useful to have "did the native library load yes/no" in the test output.  If this one fails, then you know where the problem is.  If it passes but others don't, then you have a more complicated situation to look at.